ui/admin/app/components/form/host/gcp/index.hbs

```diff
@@ -0,0 +1,112 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+
+  <Hds::Form::TextInput::Field
+    @value={{@model.type}}
+    name='type'
+    disabled={{true}}
+    as |F|
+  >
+    <F.Label>{{t 'form.type.label'}}</F.Label>
+  </Hds::Form::TextInput::Field>
+
+  {{#if @model.external_name}}
+    <Hds::Form::TextInput::Field
+      @value={{@model.external_name}}
+      @isInvalid={{@model.errors.external_name}}
+      name='external_name'
+      disabled={{true}}
+      as |F|
+    >
+      <F.Label>{{t 'form.external_name.label'}}</F.Label>
+      <F.HelperText>{{t 'form.external_name.help'}}</F.HelperText>
+      {{#if @model.errors.external_name}}
+        <F.Error as |E|>
+          {{#each @model.errors.external_name as |error|}}
+            <E.Message>{{error.message}}</E.Message>
+          {{/each}}
+        </F.Error>
+      {{/if}}
+    </Hds::Form::TextInput::Field>
+  {{/if}}
+
+  <Hds::Form::TextInput::Field
+    @value={{@model.external_id}}
+    @isInvalid={{@model.errors.external_id}}
+    name='external_id'
+    disabled={{true}}
+    as |F|
+  >
+    <F.Label>{{t 'form.external_id.label'}}</F.Label>
+    <F.HelperText>{{t 'form.external_id.help'}}</F.HelperText>
+    {{#if @model.errors.external_id}}
+      <F.Error as |E|>
```

Comment:
if its disabled, we probably wouldn't have error messages, right? 🤔

Reply:
Probably not, I clean clean this up for all the pages